tritium-metrics/src/main/java/com/palantir/tritium/metrics/executor/TaggedMetricsScheduledExecutorService.java

```diff
@@ -0,0 +1,255 @@
+        List<TaggedMetricsCallable<T>> instrumented = new ArrayList<>(tasks.size());
+        for (Callable<T> task : tasks) {
+            instrumented.add(new TaggedMetricsCallable<>(task));
+        }
+        return instrumented;
+    }
+
+    @Override
+    public void shutdown() {
+        delegate.shutdown();
+    }
+
+    @Override
+    public List<Runnable> shutdownNow() {
+        return delegate.shutdownNow();
+    }
+
+    @Override
+    public boolean isShutdown() {
+        return delegate.isShutdown();
+    }
+
+    @Override
+    public boolean isTerminated() {
+        return delegate.isTerminated();
+    }
+
+    @Override
+    public boolean awaitTermination(long timeout, TimeUnit unit) throws InterruptedException {
+        return delegate.awaitTermination(timeout, unit);
+    }
+
+    private class TaggedMetricsRunnable implements Runnable {
+
+        private final Runnable task;
+
+        TaggedMetricsRunnable(Runnable task) {
+            this.task = task;
+        }
+
+        @Override
+        public void run() {
+            running.inc();
+            try (Timer.Context ignored = duration.time()) {
+                task.run();
+            } finally {
+                running.dec();
+                completed.mark();
+            }
+        }
+    }
+
+    private class TaggedMetricsScheduledRunnable implements Runnable {
+
+        private final Runnable task;
+        private final long periodInNanos;
+
+        TaggedMetricsScheduledRunnable(Runnable task, long period, TimeUnit unit) {
+            this.task = task;
+            this.periodInNanos = unit.toNanos(period);
+        }
+
+        @Override
+        public void run() {
+            running.inc();
+            Timer.Context context = duration.time();
+            try {
+                task.run();
+            } finally {
+                long elapsed = context.stop();
+                running.dec();
+                completed.mark();
+                if (elapsed > periodInNanos) {
+                    scheduledOverrun.inc();
+                }
+                scheduledPercentOfPeriod.update((100L * elapsed) / periodInNanos);
+            }
+        }
+    }
+
+    private class TaggedMetricsCallable<T> implements Callable<T> {
+
+        private final Callable<T> task;
+
+        TaggedMetricsCallable(Callable<T> task) {
+            this.task = task;
+        }
+
+        @Override
+        public T call() throws Exception {
+            running.inc();
+            try (Timer.Context ignored = duration.time()) {
+                return task.call();
+            } finally {
+                running.dec();
+                completed.mark();
+            }
+        }
+    }
+
+    private static MetricName createMetricName(String metricName, Map<String, String> safeTags) {
+        return MetricName.builder()
+                .safeName(MetricRegistry.name("com/palantir/tritium/metrics/executor", metricName))
```

Comment:
probably don't want the long slashed name
```suggestion
                .safeName(MetricRegistry.name("scheduled-executor", metricName))
```